cornell-dti/office-hours
# To release

### Summary 
Adding calendar export and necessary bug fix to release









### Test Plan 


- Make sure calendar export works
- Make sure no duplicate notifications are sent in any case
- Make sure when switching to a different class the office hour session does not persist
### Notes 



### Breaking Changes 

None
